messenger.rst

```diff
@@ -865,6 +865,116 @@ If the message fails again, it will be re-sent back to the failure transport
+
+                    failed: 'doctrine://default?queue_name=failed'
+                    failed_other: 'doctrine://default?queue_name=failed_other' 
+
+    .. code-block:: xml
+
+        <!-- config/packages/messenger.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony
+                https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <!-- after retrying, messages will be sent to the "failed" transport 
+                by default if no "failed-transport" is configured inside a transport -->
+                <framework:messenger failure-transport="failed">
+                    <framework:transport name="async_priority_high" dsn="%env(MESSENGER_TRANSPORT_DSN)%" failure-transport="failed"/> 
+                    <!-- since no "failed_transport" is configured, the one used will be
+                    the global "failed_transport" set -->
+                    <framework:transport name="async_priority_low" dsn="doctrine://default?queue_name=async_priority_low"/>
+                    
+                    <framework:transport name="failed" dsn="doctrine://default?queue_name=failed"/>
+                    <framework:transport name="failed_other" dsn="doctrine://default?queue_name=failed_other"/>
+                </framework:messenger>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/messenger.php
+        $container->loadFromExtension('framework', [
+            'messenger' => [
+                // after retrying, messages will be sent to the "failed" transport 
+                // by default if no "failure_transport" is configured inside a transport
+                'failure_transport' => 'failed',
+
+                'transports' => [
+                    'async_priority_high' => [
+                        'dsn' => '%env(MESSENGER_TRANSPORT_DSN)%',
+                        'failure_transport' => 'failed'
+                    ],
+                    // since no failed transport is configured, the one used will be
+                    // the global failure_transport set 
+                    'async_priority_low' => [
+                        'dsn' => 'doctrine://default?queue_name=async_priority_low',
+                    ],
+                    'failed' => [
+                        'dsn' => 'doctrine://default?queue_name=failed',
+                    ],
+                ],
+            ],
+        ]);
+
+The transport with a ``failure_transport`` configuration defined, will override the global ``failure_transport``.
```

Comment:
We can maybe delete this sentence, as i've proposed the same thing above. And, we can make the code block "speak for itself" with good comments there, as you have done already :)